doc/nrf/libraries/bluetooth_services/mesh/gen_prop_cli.rst

```diff
@@ -33,94 +33,94 @@ mesh models prop instance get-all
 
 
-mesh models prop instance set <elem_idx>
+mesh models prop instance set <ElemIdx>
 	Select the Generic Property Client model instantiated on the specified element ID.
 	This instance will be used in message sending.
 	If no model instance is selected, the first model instance found on the device will be used by default.
 
-	* ``elem_idx`` - Element index where the model instance is found.
+	* ``ElemIdx`` - Element index where the model instance is found.
 
 
-mesh models prop cli-props-get <id>
+mesh models prop cli-props-get <Id>
 	Get the list of Generic Client Properties of the bound server.
 
-	* ``id`` - A starting Client Property ID present within an element.
+	* ``Id`` - A starting Client Property ID present within an element.
 
 
-mesh models prop props-get <kind>
+mesh models prop props-get <Kind>
 	Get the list of properties of the bound server.
 
-	* ``kind`` - Kind of Property Server to query. Allowed values:
+	* ``Kind`` - Kind of Property Server to query. Allowed values:
 		* ``0`` - Manufacturer Property Server.
 		* ``1`` - Admin Property Server.
 		* ``2`` - User Property Server.
 
 
-mesh models prop prop-get <kind> <id>
+mesh models prop prop-get <Kind> <Id>
 	Get the value of a property of a server.
 
-	* ``kind`` - Kind of Property Server to query. Allowed values:
+	* ``Kind`` - Kind of Property Server to query. Allowed values:
 		* ``0`` - Manufacturer Property Server.
 		* ``1`` - Admin Property Server.
 		* ``2`` - User Property Server.
 		* ``3`` - Client Property Server.
-	* ``id`` - ID of the property.
+	* ``Id`` - ID of the property.
 
 
-mesh models prop user-prop-set <id> <hex_str_val>
+mesh models prop user-prop-set <Id> <HexStrVal>
 	Set a property value of the User Property Server and wait for response.
 
-	* ``id`` - Property ID.
-	* ``hex_str_val`` - Property value.
+	* ``Id`` - Property ID.
+	* ``HexStrVal`` - Property value.
 
 
-mesh models prop user-prop-set-unack <id> <hex_str_val>
+mesh models prop user-prop-set-unack <Id> <HexStrVal>
 	Set a property value of the User Property Server without requesting a response.
 
-	* ``id`` - Property ID.
-	* ``hex_str_val`` - Property value.
+	* ``Id`` - Property ID.
+	* ``HexStrVal`` - Property value.
 
 
-mesh models prop admin-prop-set <id> <access> <hex_str_val>
+mesh models prop admin-prop-set <Id> <Access> <HexStrVal>
 	Set a property value of the Admin Property Server and wait for response.
 
-	* ``id`` - Property ID.
-	* ``access`` - User access flags for the property. Allowed values:
+	* ``Id`` - Property ID.
+	* ``Access`` - User access flags for the property. Allowed values:
 		* ``0`` - Access to the property is prohibited.
 		* ``1`` - Property may be read.
 		* ``2`` - Property may be written.
 		* ``3`` - Property may be read or written.
-	* ``hex_str_val`` - Property value.
+	* ``HexStrVal`` - Property value.
 
 
-mesh models prop admin-prop-set-unack <id> <access> <hex_str_val>
+mesh models prop admin-prop-set-unack <Id> <Access> <HexStrVal>
```

Comment:
```suggestion
mesh models prop admin-prop-set-unack <ID> <Access> <HexStrVal>
```
?

Reply:
I was a little unsure if I should go for `Id` or `ID`. But I guess I'll go for `ID`.